logging-api/src/main/java/com/cisco/oss/foundation/logging/transactions/TransactionLogger.java

```diff
@@ -419,4 +416,12 @@ private static void addInvocationPerComponent(HashMap<String, Long> mapComponent
     mapComponentInvocations.put(key, invocationComponent);
   }
 
+
+  protected void putProperty(String key , String value) {
+
+    String keyName = loggingKeys.getKeyValue(key);
+    if (keyName != ""){
```

Comment:
no need for an else clause? if not exists why not add they key as is? doesn't this mean you add something that may be component specific and without the else you're preventing this addition?